build.zig

```diff
@@ -55,7 +55,7 @@ pub fn build(b: *Builder) !void {
     });
     sdk.link(demo_wrapper, sdl_linkage);
     demo_wrapper.addModule("sdl2", sdk.getWrapperModule());
-    b.installArtifact(demo_wrapper);
+    std.Build.installArtifact(b, demo_wrapper);
```

Comment:
Can you explain that change to me? The code is equivalent, just now uses std.Build as an explicit namespace which might break in the future

Reply:
Apologies! I literally started using Ziglang yesterday and these changes likely came accidentally after trying to just get things working.

I've reverted this and fixed the README to also just use `b.installArtifact()` as well.